float64[16] O_T_EE
float64[16] EE_T_K
float64[6] K_F_ext_hat_K
float64[6] K_F_probe_K
float64 time
